SpokestackTests/TextToSpeechTest.swift

```diff
@@ -51,6 +56,22 @@ class TextToSpeechTest: XCTestCase {
         XCTAssert(delegate.didSucceed)
         XCTAssertFalse(delegate.didFail)
     }
+        
+    func testSynthesizeMarkdown() {
+        let delegate = TestTextToSpeechDelegate()
+        let config = SpeechConfiguration()
+        let tts = TextToSpeech(delegate, configuration: config)
+        
+        // successful request with markdown formatting
+        delegate.reset()
+        let didSucceedExpectation3 = expectation(description: "successful request calls TestTextToSpeechDelegate.success")
+        delegate.asyncExpectation = didSucceedExpectation3
+        let markdownInput = TextToSpeechInput("Yet right now the average age of this (52nd)[number] Parliament is (49)[number] years old, [1s] OK Boomer.</speak>", voice: .demoMale, inputFormat: .markdown)
```

Comment:
I went on a dark journey to see whether `(52nd)[number]` would work. Turns out it will, but it's not really the right way to express this in SMD. Spokestack doesn't totally support the "right" way, though, so carry on.

...however, the trailing `</speak>` is definitely wrong, and should either cause an error or some unexpected results if passed through the production synthesizer.